bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/values/BTypedesc.java

```diff
@@ -46,6 +47,13 @@ public interface BTypedesc extends BValue {
      */
     Object instantiate(Strand strand);
 
+    /**
+     * Returns the annotations specified on this type descriptor.
+     *
+     * @return annotations
+     */
+    MapValue getAnnotations();
```

Comment:
Can't we avoid having this be part of the public API since we don't support it completely atm? Can be part of `TypedescValueImpl`, right?

cc @warunalakshitha @HindujaB

Reply:
Add to the `TypedescValue` interface.